spec/validators/ingested_date_validator_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe IngestedDateValidator do
+  class IngestValidatorTest < Framework::Sheet
+    field 'An Ingested Date', :string, ingested_date: true
+  end
```

Comment:
This should be an anonymous class, so that we don't leak this into the global namespace